UnicornRender/video/source/Material.cpp

```diff
@@ -13,39 +13,45 @@ namespace unicorn
 namespace video
 {
 Material::Material()
-    : color(Color::Red())
+    : m_color(Color::Red())
     , m_isColored(true)
     , m_isWired(false)
     , m_isVisible(true)
     , m_albedo(nullptr)
 {
 }
 
-void Material::SetAlbedo(Texture const* albedo)
+void Material::SetAlbedo(std::shared_ptr<Texture> albedo)
 {
-    m_isColored = false;
     if(albedo == nullptr || !albedo->IsLoaded())
     {
-        LOG_ERROR("Setting not loaded texture to material!");
+        spdlog::get(utility::Logger::LoggerName())->error("Setting not loaded texture to material!");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.